docs/products/kafka/kafka-connect/howto/kafka-connect-debezium-tls-pg.md

```diff
@@ -0,0 +1,209 @@
+      --project $PROJECT
+    ```
+
+1. Create an external PostgreSQL integration endpoint to represent your PostgreSQL
+   database. Use the following JSON configuration and replace the placeholders:
+
+    ```bash
+    INTEGRATION_CONFIG=$(cat <<-END
+    {
+      "ssl_mode": "verify-ca",
+      "host": "<postgresql_host>",
+      "port": <postgresql_port>,
+      "user": "<postgresql_user>",
+      "ssl_client_certificate": "$(cat /path/to/your/client-cert.pem)",
+      "ssl_root_cert": "$(cat /path/to/your/ca.pem)",
+      "ssl_client_key": "$(cat /path/to/your/client-key.pem)"
+    }
+    END
+    )
+
+    avn service integration-endpoint-create \
+	   --project $PROJECT \
+	   --endpoint-name external_postgresql \
+	   --user-config "$INTEGRATION_CONFIG"
+
+   ```
+
+1. Retrieve endpoint ID of the integration endpoint you just created using this command:
+
+   ```bash
+   INTEGRATION_ENDPOINT_ID=$(
+	   avn service integration-endpoint-list --project $PROJECT \
+	   | grep external_postgresql \
+	   | awk '{print $1}'
+	  )
+   ```
+
+1. Connect PostgreSQL endpoint to Apache Kafka Connect:
+
+   ```bash
+   avn service integration-create \
+     --project $PROJECT \
+     --integration-type kafka_connect_postgresql \
+     --source-endpoint-id $INTEGRATION_ENDPOINT_ID \
+     --dest-service <kafka_connect_name>
+   ```
+
+1. Create the Debezium connector configuration to monitor your PostgreSQL database.
+   Replace the placeholders with your PostgreSQL and Apache Kafka Connect information:
+
+   ```bash
+   CONNECTOR_CONFIG=$(cat <<-END
+   {
+     "name": "debezium-postgres-connector",
+     "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+     // Omit "database.hostname", "database.port", and "database.user" if provided
+     by external integration
+     "database.password": "<postgresql_password>",
+     "database.dbname": "<database_name>",
+     "database.server.name": "<kafka_connect_server_name>",
+     "plugin.name": "pgoutput",
+     "publication.name": "debezium_publication",
+     "publication.autocreate.mode": "all_tables",
+     "endpoint_id": "$INTEGRATION_ENDPOINT_ID",
+     "transforms": "unwrap",
+     "transforms.unwrap.type": "io.debezium.transforms.ExtractNewRecordState"
+   }
+   END
+   )
+
+   avn service connector create \
+     --project $PROJECT \
+     --service <kafka_connect_name> \
+     --config "$CONNECTOR_CONFIG"
+   ```
+
+   After successfully deploying the setup:
+
+   - Apache Kafka Connect establishes a secure SSL connection with the PostgreSQL
+     database.
+   - Apache Kafka topics are automatically created, provided that the auto-creation
+     feature is enabled.
+
+      :::note
+        You must create Apache Kafka topics manually if auto-creation option is
+        not enabled before starting the connector.
+      :::
+
+   - Apache Kafka Connect streams data into topics using the naming
+   pattern: `{connector_name}.{database_name}.{table_name}`.
```

Comment:
Shouldn't this be at the root of the section instead of being in the last step? Also wondering if `deploy the setup` (and how) shouldn't be a step itself? Your call, I don't how familiar users are with this.

Reply:
Thank you, Arthur! I have replaced 'successfully deployed' with 'successfully configuring the setup' to clarify that our focus is on configuration, not deployment. I've included the steps in a dedicated verification section. Additionally, I have added a note about automatic topic creation in the 'create service' step, which is one of the first steps in the process.